server/app/views/admin/settings/AdminSettingsIndexView.java

```diff
@@ -0,0 +1,134 @@
+import j2html.tags.specialized.DivTag;
+import java.util.Optional;
+import javax.inject.Inject;
+import modules.MainModule;
+import play.twirl.api.Content;
+import services.settings.SettingDescription;
+import services.settings.SettingsManifest;
+import services.settings.SettingsSection;
+import views.BaseHtmlView;
+import views.CiviFormMarkdown;
+import views.admin.AdminLayout;
+import views.admin.AdminLayoutFactory;
+import views.style.StyleUtils;
+
+/** Displays application settings for the CiviForm Admin role. */
+public final class AdminSettingsIndexView extends BaseHtmlView {
+
+  public static final String SECTION_STYLES = "grid grid-flow-row-dense grid-cols-2 gap-8";
+  private final SettingsManifest settingsManifest;
+  private final AdminLayout layout;
+  private final CiviFormMarkdown civiFormMarkdown;
+
+  private static final ImmutableList<String> SECTIONS =
+      ImmutableList.of(
+          "Feature Flags",
+          "Branding",
+          "Custom Text",
+          "Email Addresses",
+          "Data Export API",
+          "Observability",
+          "External Services");
+
+  @Inject
+  public AdminSettingsIndexView(
+      SettingsManifest settingsManifest,
+      AdminLayoutFactory layoutFactory,
+      CiviFormMarkdown civiFormMarkdown) {
+    this.settingsManifest = checkNotNull(settingsManifest);
+    this.layout = checkNotNull(layoutFactory).getLayout(AdminLayout.NavPage.SETTINGS);
+    this.civiFormMarkdown = checkNotNull(civiFormMarkdown);
+  }
+
+  public Content render() {
+    var settingsManifestContent = div().withClasses("my-10");
+    var settingsContent =
+        div(renderSideNav(), settingsManifestContent).withClasses("flex", "flex-grow");
+    var mainContent =
+        div(settingsContent)
+            .withClasses(
+                "flex", "flex-grow", "flex-col", "px-2", StyleUtils.responsive2XLarge("px-16"));
+
+    SECTIONS.forEach(
+        sectionName ->
+            settingsManifestContent.with(
+                renderTopSection(settingsManifest.getSections().get(sectionName))));
+
+    return layout.render(layout.getBundle().addMainContent(mainContent));
+  }
+
+  private DivTag renderSideNav() {
+    var container = div().withClasses("relative", "p-6", "w-2/12");
+    var subContainer = div(h1("Settings").withClasses("text-3xl", "py-6")).withClasses("fixed");
+
+    SECTIONS.forEach(
+        sectionName ->
+            subContainer.with(
+                a(sectionName)
+                    .withClasses("block", "bold", "py-1")
+                    .withHref("#" + MainModule.SLUGIFIER.slugify(sectionName))));
+
+    return container.with(subContainer);
+  }
+
+  private DivTag renderTopSection(SettingsSection settingsSection) {
+    var container = div();
+
+    container.with(
+        h2(settingsSection.sectionName())
+            .withId(MainModule.SLUGIFIER.slugify(settingsSection.sectionName()))
+            .withClasses("text-xl font-bold mt-4 pt-4"));
```

Comment:
See comments in Slack about making the header more visually discernible from the setting names with possibly an border-bottom or some other way.

Reply:
Done